src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,43 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    /**
+     * Komentarz przykladowy
+     */
+    private static final int EMPTY = -1;
+    private static final int MAX_SIZE = 11;
```

Comment:
This constant is related to STACK_CAPACITY . Changing only one leads to an error (inconsistency).